### ISSUE TITLE
Issues with non-wheel installation

### ISSUE THREAD

**Issue by mbr** [miurahr/unihandecode#25]
When installing unihandecode as a dependency, some files seem to not get built correctly.

Direct installation works:

``` sh
$ pip install --no-use-wheel unihandecode
Collecting unihandecode
Downloading Unihandecode-0.50.tar.gz (2.8MB)
    100% |████████████████████████████████| 2.8MB 220kB/s 
Installing collected packages: unihandecode
  Running setup.py install for unihandecode
Successfully installed unihandecode-0.50
$ python
Python 2.7.5+ (default, Feb 27 2014, 19:37:08) 
[GCC 4.8.1] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from unihandecode import Unihandecoder
>>> d = Unihandecoder(lang='zh')
>>> 
```

When using `setup.py develop` on a package that has unihandecode as a dependency, it doesn't seem to build properly:

``` sh
$ python setup.py develop
[...]
Searching for unihandecode>=0.50
Reading https://pypi.python.org/simple/unihandecode/
Best match: Unihandecode 0.50
Downloading https://pypi.python.org/packages/source/U/Unihandecode/Unihandecode-0.50.tar.gz#md5=4209841643219c5434852df47b056f2d
Processing Unihandecode-0.50.tar.gz
Writing /tmp/easy_install-7I5fyz/slugger-0.2.1/temp/easy_install-XqV15N/Unihandecode-0.50/setup.cfg
Running Unihandecode-0.50/setup.py -q bdist_egg --dist-dir /tmp/easy_install-7I5fyz/slugger-0.2.1/temp/easy_install-XqV15N/Unihandecode-0.50/egg-dist-tmp-oi2uFB
zip_safe flag not set; analyzing archive contents...

Installed /tmp/easy_install-7I5fyz/slugger-0.2.1/Unihandecode-0.50-py2.7.egg
[...]
$ python
Python 2.7.5+ (default, Feb 27 2014, 19:37:08) 
[GCC 4.8.1] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from unihandecode import Unihandecoder
>>> d = Unihandecoder(lang='zh')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "build/bdist.linux-x86_64/egg/unihandecode/__init__.py", line 37, in __init__
  File "build/bdist.linux-x86_64/egg/unihandecode/unidecoder.py", line 75, in __init__
  File "build/bdist.linux-x86_64/egg/unihandecode/unidecoder.py", line 117, in _load_codepoints
  File "/home/marc/.virtualenvs/dj/local/lib/python2.7/site-packages/pkg_resources.py", line 951, in resource_stream
    self, resource_name
  File "/home/marc/.virtualenvs/dj/local/lib/python2.7/site-packages/pkg_resources.py", line 1406, in get_resource_stream
    return BytesIO(self.get_resource_string(manager, resource_name))
  File "/home/marc/.virtualenvs/dj/local/lib/python2.7/site-packages/pkg_resources.py", line 1409, in get_resource_string
    return self._get(self._fn(self.module_path, resource_name))
  File "/home/marc/.virtualenvs/dj/local/lib/python2.7/site-packages/pkg_resources.py", line 1486, in _get
    return self.loader.get_data(path)
IOError: [Errno 0] Error: 'unihandecode/unicodepoints.pickle.bz2'
```

A workaround (on an existing installation) is to remove the virtualenv's `Unihandecode-0.50-py2.7.egg` file, then install unihandecode against using the `--use-wheel` option.

I believe that this does every installation of unihandecode made using the `develop` command of `setup.py`.

**Comment by miurahr** [comment]
It should be one of release test cases.

**Comment by miurahr** [comment]
Direct installation from source works. (good),but the case which called from `setup.py develop` fails install. A difference is ... 
```
Running Unihandecode-0.50/setup.py -q bdist_egg 
```
It cannot produce working egg file.

**Comment by miurahr** [comment]
I just remember. 
When installing Egg, it does not run `setup.py build`. 
When installing wheel, it does run `setup.py build`.

It build dictionary when running `setup.py build`

When building egg file, it generate dictionary files,  but setuptools doesn't include these in egg file.

**Comment by mbr** [comment]
I've had a similar but different problem recently; I was trying to build a file and package it with a app and included the source code for it.

Build would not be run when I needed it, so I ended up tacking the build process onto a custom install command. See https://github.com/mbr/githome/blob/master/setup.py#L17

It's probably not an ideal or especially clean solution, but it may give you some ideas on how to handle it.

**Comment by miurahr** [comment]
I got it.
When just run `setup.py bdist_egg` it don't run `setup.py build`. Then it generate egg package WITHOUT dictionaries.
If we run `setup.py bdist_egg` AFTER running `setup.py build`,  it create GOOD egg package WITH dictionaries.

**Comment by miurahr** [comment]
As an interim solution for the problem, I'd like to push egg file to pypi.

**Comment by miurahr** [comment]
Now it is solved in v0.80 wheel release.
